app/src/main/res/layout/log_in_fragment.xml

```diff
@@ -32,7 +31,8 @@
         app:layout_constraintTop_toTopOf="parent" />
 
     <com.google.android.material.textfield.TextInputLayout
-        android:id="@+id/textInputLayout2"
+        android:id="@+id/emailInput"
+        android:inputType="textNoSuggestions"
         android:layout_width="220dp"
         android:layout_height="52dp"
         android:hint="@string/email"
```

Comment:
Consider naming your view elements after the view they are in, and use that in your translation strings eg: fragment_login_text_input_email. the translation string would then be @string/fragment_login_text_input_email